main/guides/zoe/contract-basics.md

```diff
@@ -1,90 +1,25 @@
 # Smart Contract Basics
 
-Before we look at how to make a contract such as the one in [the
-basic dapp](../getting-started/) in the previous section, let's cover some basics by writing a simple contract that returns a greetings message. We will simply call it _hello-world smart contract_. 
+This guide is designed to help developers understand how to write smart contracts efficiently and securely. Here, you will find detailed examples that demonstrate different functionalities and use-cases within the context of smart contracts starting from the very basics.  To follow along, verify your work, or to get help if you get stuck, you can use the https://github.com/Agoric/dapp-offer-up/ repo as a reference. Each section of the tutorial has an associated branch in the repo to act as a checkpoint.
 
-A contract is defined by a JavaScript module that exports a `start` function. For our hello-world smart contract, the declaration of `start` function looks like this:
+## Examples
 
-<<< @/../snippets/zoe/src/01-hello.js#start
+We provide three core examples to illustrate how to implement various smart contract functionalities:
 
-For the hello-world smart contract, we will have a simple `greet` function apart from `start` function. The `greet` function takes a string as a parameter (for example, name of the person calling the function) and returns a customized greeting message.
+### 1. Greetings Contract
+The **Greetings Contract** demonstrates how to create a simple contract that greets the caller. This example is perfect for understanding the basic structure and syntax of smart contracts. The relevant code is available in `tut-01-hello` branch available [here](https://github.com/Agoric/dapp-offer-up/tree/tut-01-hello).
 
-<<< @/../snippets/zoe/src/01-hello.js#greet
+### 2. State Contract
+The **State Contract** shows how to maintain a state or a variable within a smart contract. This is useful for contracts that need to store data or state across transactions. The relevant code is available in `tut-01-hello` branch available [here](https://github.com/Agoric/dapp-offer-up/tree/tut-02-state).
```

Comment:
> The content inside a link should indicate where the link goes
-- [Accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#accessibility_concerns)

```suggestion
The **State Contract** shows how to maintain a state or a variable within a smart contract. This is useful for contracts that need to store data or state across transactions. See the [`tut-01-hello` branch](https://github.com/Agoric/dapp-offer-up/tree/tut-02-state) for the relevant code.
```

Likewise for all other _here_ links.

Reply:
Thanks for pointing out. Fixed.